src/modules/Kahoot/kahoot.controller.ts

```diff
@@ -1,84 +1,24 @@
-import * as express from "express";
-import Controller from "@/interfaces/controller.interface";
-import Kahoot from "./kahoot.interface";
-import kahootModel from "./kahoot.model";
-import Response from "@/helpers/response.helper";
-import mongoose from "mongoose";
-import status from "http-status";
-import requireAuth from "@/middlewares/auth.middleware";
+import * as express from 'express';
+import kahootModel from './kahoot.model';
```

Comment:
Sửa lại cho a kahootModel -> KahootModel. Đổi toàn bộ Model theo tên như rứa nha